packages/manager/apps/octavia-load-balancer/src/components/at-internet/index.js

```diff
@@ -0,0 +1,27 @@
+import angular from 'angular';
+import ovhManagerAtInternetConfiguration from '@ovh-ux/manager-at-internet-configuration';
+import { registerAtInternet } from '@ovh-ux/ng-shell-tracking';
+
+import TRACKING from './at-internet.constant';
+
+const moduleName = 'publicCloudAtInternet';
```

Comment:
**nitpick:** Since this is `octavia-load-balancer` app, it would be more appropriate to rename the module.
```suggestion
const moduleName = 'octaviaLoadBalancerAtInternet';
```